hypervisor/arch/x86/Kconfig

```diff
@@ -50,7 +50,7 @@ config LOG_BUF_SIZE
 
 config LOG_DESTINATION
 	int "Bitmap of consoles where logs are printed"
-	default 3
+	default 7
```

Comment:
Why we change the default log level ?  Turn on NKP should be some time later..

Reply:
LOG_DESTINATION is not the default log level.
The npk_log is disabled by default, and can be enabled by hypercall in runtime.